### ISSUE TITLE
the threshold in 3DRegNet+U

### ISSUE THREAD

**Issue by MrZ19** [goncalo120/3DRegNet#4]
Hi, in 3DRegNet + U experiment, what is the threshold value to classify the correspondences into correct/incorrect ones

**Comment by goncalo120** [comment]
Hi!
In the 3DRegNet part, we select the inliers by choosing the logits that are greater than 0:
https://github.com/goncalo120/3DRegNet/blob/01f8924801281aad9bf706d35f0baf18118763b3/test.py#L131
